### ISSUE TITLE
syntax highlighting

### ISSUE THREAD

**Issue by mrloop** [wting/gitsessions.vim#14]
Loading a vim session explicitly with `vim -S ~/.vim/sessions/path/to/my/gitsessions/vim/session` results in expected syntax highlighting.
Loading a vim session automatically with `vim` results in no syntax highlighting. Reloading the file(s) with `:e` and I get the syntax highlighting expected.

**Comment by wting** [comment]
Yeah there was an attempted fix (#12) but that PR broke gitsessions completely. Either the session loaded with syntax highlighting or didn't load at all. I figured that partially loading w/o syntax highlighting was a better alternative than complete breakage.

**Comment by mrloop** [comment]
Thanks @wting, I'm going to try running the attempted fix #12 and see how it goes. I've usually got 6 to 8 panes open and no syntax highlighting is a show stopper for me.

**Comment by simonwahlgren** [comment]
I'm using a custom script but I had the same issue when restoring the sessions, this answer solved my problem: http://stackoverflow.com/a/10525050

**Comment by wting** [comment]
Thanks for the comment @simonwahlgren, but the [nested call](https://github.com/wting/gitsessions.vim/pull/12/files) that fixed the syntax highlighting issue broke session reloading.

**Comment by cesarizu** [comment]
Just for the record, the fix from #12 fixed the syntax highlighting for me. I think it is because I'm only saving the buffers (`set sessionoptions=buffers`).
